### PR TITLE
Update simple example and usage example in readme

### PR DESCRIPTION
**Pull request by jtrip**
Update simple example and usage example in readme, additional line after imports for pep 8.

As mentioned in #15 the simplest test example was blank although there was a good example in the readme file. I update the simple test file to match the usage example in the readme file and also put the display updating call in a `while True:` loop so a user will be sure to see the output.

I also added an additional blank line after the import statements in that file for pep 8 compliance. I also then copied that code to the usage example in the readme so that the update is reflected there.

I built the docs to verify the changes were reflected there, they were and I saw no other relevant aspect of the docs that needed updating. Even though the simple test file was blank the docs were already pointing to it, probably because there are so many of these wonderful libraries. 😄 

If the pep 8 line is not welcome I can remove that and resubmit.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -36,11 +36,13 @@ For a board with a built-in display.
     import terminalio
     from adafruit_display_text import label
 
+
     text = "Hello world"
     text_area = label.Label(terminalio.FONT, text=text)
     text_area.x = 10
     text_area.y = 10
-    board.DISPLAY.show(text_area)
+    while True:
```
Comment (ladyada) [inline README.rst]: show and then while True: pass

File: examples/display_text_simpletest.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import board
+import terminalio
+from adafruit_display_text import label
+
+
+text = "Hello world"
+text_area = label.Label(terminalio.FONT, text=text)
+text_area.x = 10
+text_area.y = 10
+while True:
```
Comment (ladyada) [inline examples/display_text_simpletest.py]: ditto

**Comment by jtrip** [comment]
Ah yes, sorry I misunderstood you the first time you described it. Having it do less makes sense to me!  
That should be good now.

**Comment by ladyada** [comment]
@makermelissa please review when you can

**Pull request merged**